src/Compilers/Core/Portable/PEWriter/PeWriter.cs

```diff
@@ -104,6 +104,7 @@ internal static class PeWriter
                 }
 
                 nativePdbWriterOpt.WriteRemainingEmbeddedDocuments(mdWriter.Module.DebugDocumentsBuilder.EmbeddedDocuments);
```

Comment:
>WriteRemainingEmbeddedDocuments [](start = 35, length = 31)

I don't think this is necessary anymore - embedded sources are added to the set of all debug documents on the document builder in `CreateDebugDocuments`. If we make sure we add an entry for each document then it follows that all embedded documents will also be added.

Also, once we remove this call we can remove `EmbeddedDocuments` array from `DebugDocumentsBuilder`.